src/components/App/App.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable react/jsx-filename-extension */
+import React from 'react';
+
+import { GlobalStyles } from '../../GlobalStyles.styles';
+import HomePage from '../../pages/Home/Home.page';
+
+function App() {
+  return (
+    <div className="App">
```

Comment:
Are this classname and div element necessary? I couldn't find the classname reference, but if you need a parent element you can use a react fragment, I like the shorthand https://reactjs.org/docs/fragments.html#short-syntax